sql-cli/sql_cli/connections.py

```diff
@@ -27,11 +27,6 @@ def convert_to_connection(conn: dict[str, Any], data_dir: Path) -> Connection:
     if connection["conn_type"] == "sqlite" and not os.path.isabs(connection["host"]):
         # Try resolving with data directory
         resolved_host = data_dir / connection["host"]
-        if not resolved_host.is_file():
-            raise FileNotFoundError(
-                f"The relative file path {connection['host']} was resolved into {resolved_host}"
-                " but it's a failed resolution as the path does not exist."
-            )
```

Comment:
Ideally not, because the sqlite `test_connection` will check that. But I just noticed that this is currently being used in generate and run too. Won't that fail on run time in "run"? I think we should run handle this. WDYT @dimberman ?

Reply:
Yeah ok I can see where allowing it in run should be ok. As long as there's an expected flow (e.g. users validate their connections separately), I can see where this is unecessary.